samcli/lib/init/templates/cookiecutter-aws-sam-hello-golang/tests/test_bake_project.py

```diff
@@ -19,61 +19,58 @@ def inside_dir(dirpath):
 
 
 def test_project_tree(cookies):
-    result = cookies.bake(extra_context={'project_name': 'test_project'})
+    result = cookies.bake(extra_context={"project_name": "test_project"})
```

Comment:
all these changes are courtesy of `black` running automatically on these changed files.

Reply:
They were explicitly ignored before. You need to update the path in the pyproject.toml file

Reply:
yeah I updated the path *after* black made the changes. Any risk you see with this change?